packages/aggregator/src/createAggregator.js

```diff
@@ -6,7 +6,7 @@
 import { DataBroker } from '@tupaia/data-broker';
 import { Aggregator } from './Aggregator';
 
-export const createAggregator = (AggregatorClass = Aggregator, ...additionalParams) => {
-  const dataBroker = new DataBroker();
+export const createAggregator = (AggregatorClass = Aggregator, context, ...additionalParams) => {
```

Comment:
This will break existing usages of `createAggregator`, eg in [web-config-server](https://github.com/beyondessential/tupaia/blob/dev/packages/web-config-server/src/apiV1/DataAggregatingRouteHandler.js#L18)

Reply:
You are right, how would you feel about merging this code in to the feature branch even if it breaks web-config-server, with a plan to refactor web-config-server over to using this type of entity-aggregation before we merge the feature branch into `dev`?

>Side note: to be honest, I'm not really sure why we have to use createAggregator and not just instantiate Aggregator objects ourselves. There would be a motivation for establishing this factory pattern, but I can't remember now... Just creating Aggregator instances would probably make things simpler!

I totally agree with this and would like to do it as part of the above mentioned refactor?

Reply:
Agreed! Although I believe we should not migrate `web-config-server` to use the new "entity server in aggregator" functionality as part of a PSSS release. This would unnecessarily increase the regression testing scope.

Fine with `web-config-server` being broken in `psss-phase-2` for now